cmd/definitions.go

```diff
@@ -149,7 +149,7 @@ type ConfigurationFile struct {
 	Src          string `yaml:"src"`
 }
 
-// Kubernetes configurations
+// KubeConfig
```

Comment:
There should really be a Kubernetes configuration section separate from endpoints like metrics that are not k8s specific.